pkgs/applications/graphics/sane/backends/airscan/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, avahi, libsoup, libjpeg
+, sane-backends, glib, ctags }:
+stdenv.mkDerivation rec {
+  pname = "sane-airscan";
+  version = "0.9.16";
+
+  nativeBuildInputs = [ pkg-config ctags ];
+  buildInputs = [ avahi libsoup libjpeg sane-backends glib ];
+
+  makeFlags = [
+    "PREFIX=\${out}"
+    "PKG_CONFIG=${pkg-config}/bin/pkg-config"
+    "LIBDIR=/lib"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "alexpevzner";
+    repo = pname;
+    rev = version;
+    sha256 = "05zqq7majj3li3ini9sqb79lcj5fxddr3ycmj1yrfbch6781y7yq";
+  };
+
+  fixupPhase = ''
```

Comment:
Avoid overriding phases. Instead, use `preFixup` or `postFixup`.

Reply:
Thanks, missed that.